app/controllers/books_controller.rb

```diff
@@ -16,20 +16,6 @@ def show
     raise PageNotFound unless @book
   end
 
-  def commands
-    @related = {}
-    ri = RelatedItem.where(:content_type => 'reference', :related_type => 'book')
-    ri.each do |item|
-      cmd = item.name.gsub('git-', '')
-      if s = Section.where(:slug => item.related_id).first
-        @related[cmd] ||= []
```

Comment:
Is the `related` stuff even active in the site?

Reply:
No, it's not. We'd have to run `rake related` to do the indexing, and we haven't done that in a long time (there were some "related" records lingering in the database, but they got wiped during one of the many transitions).

It actually does to work if you:

  1. Run with `UPDATE_TOKEN=foo script/rails server`

  2. Populate the database with `UPDATE_TOKEN=foo CONTENT_SERVER=http://localhost:3000 rake related`

  3. Look at book pages, which now sometimes cross-reference mentioned commands. Manpages don't seem to mention each other though.

Given that the "related" sidebar has been missing for a year without anybody noticing, and given that it'd take some work to clean up the update rake task to work directly on the database, I'm inclined to just rip the whole thing out.

Reply:
Also, the presence of `related` makes the existence of the `books/commands` endpoint more sensible. It wasn't just a list of commands, but it actually cross-referenced the book chapters (or it would have if we had run `rake related` regularly). I'm still of the opinion that we're better off just pointing to the `/docs` endpoint.